tests/tests.py

```diff
@@ -1883,16 +1892,16 @@ def test_update(self):
 
         version.delete()
 
-    def test_delete(self):
-        version = self.jira.create_version('To be deleted', self.project_b,
+    def test_delete_version(self):
+        version_str = "test_delete_version:" + self.manager.jid
+        version = self.jira.create_version(version_str, self.project_b,
                                            releaseDate='2015-03-11',
                                            description='not long for this world')
-        myid = version.id
         version.delete()
-        self.assertRaises(JIRAError, self.jira.version, myid)
+        self.assertRaises(JIRAError, self.jira.version, version.id)
 
-    def test_version_expandos(self):
-        pass
+    # def test_version_expandos(self):
```

Comment:
It was empty and without any information regarding what it was supposed to go there.